git-coco

```diff
@@ -164,6 +164,31 @@ pr() {
     echo "https://console.aws.amazon.com/codesuite/codecommit/repositories/${repo}/pull-requests/${id}/changes"
 }
 
+tag(){
+    local repo arn
+
+    repo="$1"
+    tags="$2"
```

Comment:
This should also be a "`local`" variable

Reply:
Added!